tests/dummy/app/helpers/gradient-area.js

```diff
@@ -0,0 +1,26 @@
+/**
```

Comment:
This should be inside the addon itself as a helper available for general use. Right now its in your dummy app only. `/test/dummy/app`